rsdp/src/lib.rs

```diff
@@ -169,6 +169,14 @@ impl Rsdp {
         Ok(())
     }
 
+    pub fn signature(&self) -> [u8; 8] {
+        self.signature
+    }
+
+    pub fn checksum(&self) -> u8 {
+        self.checksum
+    }
+
```

Comment:
Rather than defining getters for `signature` and `checksum`, the fields could be made public. (Maybe there is some reason not to, though, as I see there are already getters for `revision` and `rsdt_address`, which puzzles me a bit.)

Reply:
Maybe! I thought getters were only necessary for `rsdt_address`, `length` and `xsdt_address` because they have alignments greater than 1, but it could be for consistency.